bundles/CoreBundle/Migrations/Version20221005122653.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20221005122653 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add fulltext index to search_backend_data table';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql("CREATE FULLTEXT INDEX search_backend_data_fulltext_index ON search_backend_data(`data`, `properties`, `fullpath`);");
```

Comment:
I think we should replace the existing index here, otherwise we're more or less doubling the size of the full-text index. So actually we need to drop the existing index here as well.